lib/cocoapods/validator.rb

```diff
@@ -896,6 +906,10 @@ def podfile_from_spec(platform_name, deployment_target, use_frameworks = true, t
       podspec  = file.realpath
       local    = local?
       urls     = source_urls
+
+      additional_podspec_pods = Dir.glob(external_podspecs || '')
```

Comment:
Should this be `external_podspecs ? Dir.glob(external_podspecs) : []`?

Reply:
I originally did this, but it felt a bit weird re-defining the same variable to be a different type. I'm happy to change this if it's a blocker though 👍

Reply:
doh - forgot to push this, hah

Reply:
dont care so much about the variable name, but I dont like having a call to `Dir.glob('')`

Reply:
Ah, that makes sense - will fix

Reply:
Fixed 👍